### ISSUE TITLE
z_init_postback_forms called for all incoming messages

### ISSUE THREAD

**Issue by mmzeeman** [zotonic/zotonic#1159]
Every time a message arrives ```z_init_postback_forms``` is called. This function initialises possibly placed forms on the page by looking if there is an uninitialised form. 

It is not needed to call this function for message acks, session, reload messages, or custom delegates.

We should probably trigger this function in a different way so it is called less often. When a websocket is open it is currenly called every 20 seconds.

**Comment by mworrell** [comment]
Good point. At least we should only call it for javascript payloads.
Is there a way to see if javascript changed the DOM?

**Comment by mmzeeman** [comment]
It is possible to detect changes with http://caniuse.com/#search=mutation%20observer. From IE11 this works. Maybe there is a shim/polyfill for IE9 an 10.

During a recent IE bughunt I implemented this via a trigger mechanism with timeouts. *I got permission denied errors on this function, but that was caused by something else.* This also worked, but was more complex than I liked so I did not push it.
